config/settings.yml

```diff
@@ -0,0 +1,6 @@
+---
+:manageiq_rubygems:
+  :s3_access_key:
+  :s3_secret_key:
```

Comment:
Minor, but I would name these s3_access_key_id and s3_secret_access_key to match the terminology that aws uses.